tests/test_api.py

```diff
@@ -137,6 +138,65 @@ def test_file_name_is_deprecated(self):
             warnings[0].message.args[0]
         )
 
+    def test_mode_once(self):
+        temp_dir = os.path.join(FILE_DIR, 'perf_files/')
+        with temporary_path(temp_dir):
+
+            with record(path='perf_files/api/'):
+                caches['default'].get('foo')
+
+            full_path = os.path.join(
+                FILE_DIR,
+                'perf_files',
+                'api',
+                'test_api.perf.yml',
+            )
+            assert os.path.exists(full_path)
+
+    def test_mode_none(self):
+        with mock.patch('django_perf_rec.settings.PerfRecSettings.get_setting') as getter:
```

Comment:
use django's `override_settings` as a decorator on the test methdo (and below). It also means you aren't breaking any future settings 😄